app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -82,6 +82,7 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
           await github.repositories.createStatus(slug, pr.head.sha, request);
           update.status = latestStatus;
           update.updates += 1;
+          update.updateTimeMillis = DateTime.now().millisecondsSinceEpoch;
```

Comment:
A test for this line?

Reply:
Have added corresponding test for `/push_build_status_to_github` for the same functionality.

Since we are missing the test for this API, I have created an issue: https://github.com/flutter/flutter/issues/58613 to track all missing tests.

So will skip the test here.

Reply:
Nit: instead of adding all the tests, we could just copy a small portion of `push_build_status_to_github_test.dart` to `push_engine_status_to_github_test.dart` to test this line.

Reply:
Added.